### PR TITLE
Brainrake day01

### PR DESCRIPTION
**Pull request by brainrake**
Solution to Week01/Day01

### CODE REVIEW

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs
Comment (smorci) [inline Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs:1]: Very cool, but it is not part of the requirement, please remove. With that also remove dependencies please.

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html
Comment (smorci) [inline Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html:1]: Very cool, but it is not part of the requirement, please remove.

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+#[cfg(test)]
+mod tests {
+    use crate::years_to_days;
+    use num::BigUint;
+    use std::str::FromStr;
+
+    #[test]
+    fn test_0() {
```
Comment (smorci) [inline Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/lib.rs:36]: Same applies here as in [this PR comment](https://github.com/Smorci/100-Days-Of-Rust/pull/4/files#r1498273676)

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/README.md
Comment (smorci) [inline Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/README.md:1]: Like the README, nice. We should make a template for creating cargo packages.

**Review by smorci — CHANGES_REQUESTED** [review]
Structured and nice

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs:1]: This is test code and development tooling, not part of the application functionality. The deps should be dev-dependencies, but I couldn't get it to run that way. Maybe I can move it to an [example](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples) which has separate deps iirc.

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html:1]: As above.

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/lib.rs (hunk 1 above)
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/lib.rs:36]: Same response.

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/README.md
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/README.md:1]: :+1:

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+use num::BigUint;
+use std::io;
+use convert_ages_to_days_brainrake::years_to_days;
+
+fn main() {
```
Comment (brainrake) [inline Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/main.rs]: maybe this should be a cargo `example` too, pending discussion on slack

**Review by smorci — APPROVED** [review]

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/main.rs (hunk 2 above)
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/main.rs]: decided against it, this is fine

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html:1]: Didn't fix it yet.

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs:1]: Didn't fix it yet

**Review by botiboti — APPROVED** [review]
nice, I like how clean it is

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/src/bin/web.rs:1]: Moved to `examples/web/`.

File: Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html
Comment (brainrake) [reply Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_brainrake/index.html:1]: Moved to `examples/web/`.

**Pull request merged**